tests/testSimpleShape.py

```diff
@@ -63,7 +63,7 @@ def checkMoments(self, dEllipseCore, dCenter, wEllipseCore, wCenter):
         image = lsst.afw.image.MaskedImageF(self.bbox)
```

Comment:
Not here, but near the top, `os` is imported but not used. (If you have not already done so, I suggest you configure your editor to live-lint python files; flake8 and pylint are both good and most editors can be configured to run one of them).